app.js

```diff
@@ -0,0 +1,60 @@
+// empty array to store the books
+let books = [];
+
+//  function to add a new book to the collection, with title and author
+function addBook(title, author) {
+  const book = {
+    title,
+    author,
+  };
+  books.push(book);
+}
```

Comment:
- [ ] When I add books of the same title and then delete one of them, all books of the same title get deleted. This is a result of not setting a unique ID for books when creating one. Setting an ID for a book helps you delete a book by it ID so it's only the book with that ID is removed in the filter method for removing a book.
- [ ] Kindly work on setting a random ID for a book when it's created so only that book is removed by its ID. 👍🏽